libraries/AP_Notify/NavigatorLED.cpp

```diff
@@ -0,0 +1,86 @@
+/*
+   This program is free software: you can redistribute it and/or modify
+   it under the terms of the GNU General Public License as published by
+   the Free Software Foundation, either version 3 of the License, or
+   (at your option) any later version.
+
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU General Public License for more details.
+
+   You should have received a copy of the GNU General Public License
+   along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+// This driver is for the neopixel on the Blue Robotics Navigator board
+// designed for the Raspberry Pi 4. The neopixel data in is connected to
+// the Raspberry Pi's MOSI pin. The clock, chip select, and MISO pins are
+// not used. The data is sent to the neopixel in 24 'SPI bytes', where each
+// spi byte is formatted to appear as a single bit of data to the neopixel.
+
+#include <AP_HAL/AP_HAL.h>
+#include "AP_Notify/AP_Notify.h"
+#include "NavigatorLED.h"
+
+#define NEOPIXEL_LED_LOW    0x18
+#define NEOPIXEL_LED_MEDIUM 0x40
+#define NEOPIXEL_LED_HIGH   0xFF
+#define NEOPIXEL_LED_OFF    0x00
+
+#define LED_T0 0b11000000
+#define LED_T1 0b11110000
+
+extern const AP_HAL::HAL& hal;
+
+NavigatorLED::NavigatorLED() :
+    RGBLed(NEOPIXEL_LED_OFF, NEOPIXEL_LED_HIGH, NEOPIXEL_LED_MEDIUM, NEOPIXEL_LED_LOW)
+{
+}
+
+bool NavigatorLED::hw_init()
```

Comment:
You need to rebase due to #17035 and just call this `init` rather then `hw_init`

Reply:
done!